src/graphql/packet.rs

```diff
@@ -51,6 +53,12 @@ struct Packet {
     packet: String,
 }
 
+#[derive(SimpleObject, Debug)]
+struct Pcap {
+    request_time: DateTime<Utc>,
+    parsed_pcap: Vec<String>,
```

Comment:
Why is this a vector? In your code, this looks having just one item all the time.

Reply:
When we first started, the code in which Bly provided was Vec<String> at first ,so just in case, I did not want to change it since I wasn't sure if it would break something.

Reply:
Done